src/rouge/worker/worker.py

```
@@ -157,26 +246,70 @@ def execute_workflow(self, issue_id: int, description: str) -> bool:
+        """
+        try:
+            if status == "patch pending":
+                _, success = self._execute_patch_workflow(issue_id)
+            else:
+                _, success = self._execute_main_workflow(issue_id, description)
+            return success
 
         except subprocess.TimeoutExpired:
-            self.logger.error(f"Workflow timed out for issue {issue_id}")
-            update_issue_status(issue_id, "pending", self.logger)
+            self.logger.exception(f"Workflow timed out for issue {issue_id}")
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)